spec/system/admin/users_spec.rb

```diff
@@ -104,6 +104,17 @@
           expect(page).to have_content("Account updated")
           expect(page).to have_unchecked_field "Disabled"
         end
+
+        it "should toggle the api key generation view" do
+          expect(page).to have_checked_field "Show API key view for user"
+          uncheck "Show API key view for user"
+          expect(page).to have_content("Show API key view has been changed!")
+          expect(page).to have_unchecked_field "Show API key view for user"
+
+          check "Show API key view for user"
+          expect(page).to have_content("Show API key view has been changed!")
+          expect(page).to have_checked_field "Show API key view for user"
+        end
```

Comment:
I think that we should assert the attribute of the user after each change, too.

Reply:
All right!